tests/embedders/test_count_embedder.py

```diff
@@ -1,18 +1,35 @@
 """CountEmbedder tests."""
 from contextlib import nullcontext as does_not_raise
-from typing import Any, List, Union
+from typing import TYPE_CHECKING, Any, List, Union
 
-import geopandas as gpd
 import pandas as pd
 import pytest
 from pandas.testing import assert_frame_equal, assert_index_equal
 
 from srai.constants import REGIONS_INDEX
 from srai.embedders import CountEmbedder
 
+if TYPE_CHECKING:  # pragma: no cover
+    import geopandas as gpd
```

Comment:
Why like that? We have geopandas in our main dependencies

Reply:
Ruff triggered an error because geopandas is used here only for type checking.